src/pkcs1v15.rs

```diff
@@ -214,6 +219,162 @@ fn non_zero_random_bytes<R: RngCore + CryptoRng>(rng: &mut R, data: &mut [u8]) {
     }
 }
 
+pub struct RsaPkcs1v15SigningKey {
```

Comment:
These names seem to run afoul of the "no module prefix" convention:

https://rust-lang.github.io/rfcs/0356-no-module-prefixes.html

i.e. the full name is:

`rsa::pkcs1v15::RsaPkcs1v15SigningKey`

Instead it could just be:

`rsa::pkcs1v15::SigningKey`

This allows someone to import `rsa::pkcs1v15` and get access to:

- `pkcs1v15::SigningKey`
- `pkcs1v15::VerifyingKey`
- `pkcs1v15::Signature`

...rather than having to import three types that all start with the module name.